### ISSUE TITLE
Making executable

### ISSUE THREAD

**Issue by ChobinhoNL** [python-openxml/python-docx#983]
Hey! 

Somehow using pyinstaller isn't working with python-docx module. I keep getting an error saying how python-docx is not included in the executable file. I saw another person with this issue: https://stackoverflow.com/questions/35642322/pyinstaller-and-python-docx-module-do-not-work-together

That sollution is not working for me, unfortunatly.

**Comment by ChobinhoNL** [comment]
Bump

**Comment by gh421** [comment]
yes same.  
did you sort the issue?  
I tried with aspose.words to replace python-docx and docx-mailmerge but the pyinstaller created exe crashes and I get a python error message.

**Comment by ChobinhoNL** [comment]
> yes same. did you sort the issue? I tried with aspose.words to replace python-docx and docx-mailmerge but the pyinstaller created exe crashes and I get a python error message.

Nope, no solution. What I did is converted .docx into pdfs and printed what I needed to alter on top of the pdf. Ugly solution, dependant on howo the document looks like. At least now my wife can work with it. I did not get the .exe to work on her laptop with python-docx.

**Comment by gh421** [comment]
Do you mean it creates the exe and when you run it, you immediately get an error before you even see the screen load? Or when your program tries to create or modify a docx?  If you see the main window screen, that could be a different issue to what I was having.

**Comment by ChobinhoNL** [comment]
> Do you mean it creates the exe and when you run it, you immediately get an error before you even see the screen load? Or when your program tries to create or modify a docx? If you see the main window screen, that could be a different issue to what I was having.

I don't remember, it was quite a while ago. See the link I posted earlier, that was the issue I was having.

**Comment by gh421** [comment]
Just an update, I did a fresh install of python3.6 with the standard idle ide on my notebook tonight.  I copied the ui file and the python script (py) over to the laptop.  I ran it in python and added installed only the models it needed, after upgrading pip (like v21 now, the original was like v3x). 

Anyway, after installing PyQt5, docx-mailmerge, python-docx, and finally pillow (ie with pip install packagename)  the install created fine and the EXE now runs without issue with python-docx and docx-mailmerge,  The UI doesn't look the same though, slightly different even using the same QSS file.  The only thing I didn't install on the laptop was QtDesigner5.   I am wondering if the problem has to do with pip not uninstalling all files and leaving some resident directories that mess with pyinstaller or possibly some compatibility issue with QT5 designer as it seems to uses different libraries.   I am pretty sure it's pip not uninstalling all files more than likely.   

So if your program exe isn't even launching your UI, try installing on a clean version of python3.6 (easy to switch to another ide if it works) and manually copy your files (like yourapp.py and yourapp.ui) over.  Run your program and just install what it needs when it errors out.

**Comment by scanny** [comment]
Closed as inactive.
